spark/src/test/scala/org/apache/spark/sql/delta/MergeIntoSchemaEvolutionSuite.scala

```diff
@@ -502,7 +502,7 @@ trait MergeIntoSchemaEvolutionBaseTests {
   testEvolution(s"case-sensitive insert")(
     targetData = Seq((0, 0), (1, 10), (3, 30)).toDF("key", "value"),
     sourceData = Seq((1, 1), (2, 2)).toDF("key", "VALUE"),
-    clauses = insert("(key, value, VALUE) VALUES (s.key, s.value, s.VALUE)") :: Nil,
+    clauses = insert("(key, value) VALUES (s.key, s.value)") :: Nil,
```

Comment:
There were two different possible errors here. It can throw a different message dependent on whether we batch. I removed the ambiguity so that the error is always the same.

Reply:
I've added a TODO for the other test, since it's an unrelated new test and we didn't have coverage for it before.